### PR TITLE
Fixed issue identify tuples without brackets

### PR DESCRIPTION
**Pull request by harshsingh-24**
Fix the issue #1577

### CODE REVIEW

File: tests/reference/run_dbg-test_assert_01-2f34744.stderr [hunk 1]
```diff
@@ -1,5 +1,2 @@
-  File "tests/runtime_errors/test_assert_01.py", line 4
-    test()
-  File "tests/runtime_errors/test_assert_01.py", line 2
-    assert False
+warning: The `runtime stacktrace` is not enabled. To get the stacktraces, re-build LPython with `-DWITH_RUNTIME_STACKTRACE=yes`
```
Comment (faze-geek) [inline tests/reference/run_dbg-test_assert_01-2f34744.stderr:1]: Hi, are you using `MSVC `on a windows device to build ?

Comment (faze-geek) [reply tests/reference/run_dbg-test_assert_01-2f34744.stderr:1]: I doubt that this change should not happen here. You should run `python run_tests.py -u --skip-run-with-dbg`

**Comment by czgdp1807** [comment]
@Thirumalai-Shaktivel Requesting you to review this as this is a parser related change.

**Comment by harshsingh-24** [comment]
Hey @faze-geek and @Thirumalai-Shaktivel. I have fixed all the build issues associated with Linux OS. When I executed `run_tests.py - u` it modified those files and removed the Assertions from them. Finally, my first PR of this org is ready for review ✨😁

File: tests/parser/tuple1.py [hunk 2]
```diff
@@ -35,3 +35,5 @@ def test():
     x,
     y,
 )
+
+t : tuple[i32, i32] = 2,3    # Without bracket annotated assignment of tuples
```
Comment (Thirumalai-Shaktivel) [inline tests/parser/tuple1.py:39]: Can you please add a new line at the end of the file?

File: tests/reference/run_dbg-test_assert_01-2f34744.stderr [hunk 3]
```diff
@@ -2,4 +2,4 @@
     test()
   File "tests/runtime_errors/test_assert_01.py", line 2
     assert False
-AssertionError
+AssertionError
```
Comment (Thirumalai-Shaktivel) [inline tests/reference/run_dbg-test_assert_01-2f34744.stderr:5]: Remove all these `*.stderr` file changes.

**Comment by Thirumalai-Shaktivel** [comment]
Awesome, thanks for this fix!
I have left some comments above.

**Comment by Thirumalai-Shaktivel** [comment]
> I would like to modify the documentation regarding this. Can you point me to the correct directory?

I think you can write this intruction in README.md: https://github.com/lcompilers/lpython#windows-1

File: tests/parser/tuple1.py (hunk 2 above)
Comment (harshsingh-24) [reply tests/parser/tuple1.py]: Added but last newline is not visible from github.

File: tests/reference/run_dbg-test_assert_01-2f34744.stderr (hunk 3 above)
Comment (harshsingh-24) [reply tests/reference/run_dbg-test_assert_01-2f34744.stderr]: Done

**Comment by Thirumalai-Shaktivel** [comment]
Cool, let's merge this PR. Thank you!

**Comment by certik** [comment]
@harshsingh-24, thanks for the fix!

File: README.md [hunk 4]
```diff
@@ -103,6 +103,17 @@ inst\bin\lpython examples\expr2.py -o a.out
 a.out
 ```
 
+- After you update a test case file, you also need to update all the reference results associated with that test case:   
+
+```
+python run_tests.py -u --skip-run-with-dbg
+```
```
Comment (certik) [inline README.md:110]: The `--skip-run-with-dbg` flag should only be used on Windows, on Linux/macOS it should not be used.

Comment (faze-geek) [reply README.md:110]: Yes, more specifically for the `MSVC` build setup. On windows with WSL, `./run_tests.py -u ` runs just fine.

Comment (certik) [reply README.md:110]: I see it's in the Windows section of the README. I guess that's fine. We should eventually refactor it into a dedicated section and show commands for all platforms.

**Pull request merged**